source/release-notes.txt

```diff
@@ -10,6 +10,27 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.44.7
+----------------
+
+*Released November 20, 2024*
+
+New Features:
+
+- Adds an expandable json editor
```

Comment:
if you wanna link [COMPASS-4635](https://jira.mongodb.org/projects/COMPASS/issues/COMPASS-4635)

Reply:
meh, on second thought that ticket description isn't very related to this change